### PR TITLE
Search: Hide Authorization header in EP Debug Bar request log

### PR DESCRIPTION
**Pull request by brandon-m-skinner**
[PLAT-975]

## Description

Added and required 'ep_get_query_log' function to override queries and filter out the Authorization header so it's not shown in the ElasticPress debug bar/QM panel.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [x] This change has relevant unit tests (if applicable).
- [x] This change has relevant documentation additions / updates (if applicable).

## Steps to Test
1. Make sure you constants are set locally for Search(including relevant Search-specific username and password constants).
2. Do a search locally. Check the headers of the action in the EP debug bar. You should see an `Authorization` header.
3. Pull down PR.
4. Do a search locally(again). Check the headers of the action in the EP debug bar. You shouldn't see the `Authorization` header anymore.

[PLAT-975]: https://vipjira.atlassian.net/browse/PLAT-975

### CODE REVIEW

**Comment by brandon-m-skinner** [comment]
@nickdaugherty Didn’t see anything in QM for Authorization headers. I might have missed it, but I didn’t see it.

File: search/includes/functions/ep-get-query-log.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+require_once __DIR__ . '../../../elasticpress/elasticpress.php';
```
Comment (mjangda) [inline search/includes/functions/ep-get-query-log.php]: Aside: these traversals are going to bite us at some point and we need to stop doing them. In a future PR probably best to define a constant called `VIP_SEARCH__DIR__` and switch to relative paths from there.

/cc @nickdaugherty @chrean @hanifn

**Comment by brandon-m-skinner** [comment]
Found the QM EP Debug stuff. It's the same panel, so the changes worked for both.

**Comment by brandon-m-skinner** [comment]
r820-stacks

**Pull request merged**